_posts/2020-02-24-transit-gateway.md

```diff
@@ -0,0 +1,263 @@
+tags: devops networking transitgateway aws
+author: Marcelo Aravena
+---
+
+# A DevOps Guide to AWS Transit Gateway
+![alt text](/img/transit-plane-gateway.png)
+### On-Premise to Cloud Network Infrastructure Modernization
+
+Have you ever booked a flight and found yourself making a stopover somewhere just to embark on a different flight to reach your final destination? That stopover being an airport hub in some city you had to transit through in order to get on a plane which knows or can make the final non-stop flight to your destination.  That stopover is the manifestation of AWS Transit Gateway!
+
+It can be quite common these days to find yourself in a hybrid networking environment with your head in the clouds and your feet madly running around on earth maintaining IT Infrastructure with connectivity North-South, East-West, or you may have ended up with an expanding VPC-Peering or VPC Sharing infrastructure that resembles last nights spaghetti bolognese!  To help remove some of the complexity and overhead of managing a hybrid network infrastructure with AWS, you can modernize your networking infrastructure with AWS Transit Gateway, a Hub amd Spoke networking model making it easy to manage, monitor and maintain.
+
+As you start making more use of AWS Cloud resources and services to help your company grow and scale, complexity will also increase.  AWS Transit Gateway's main advantage is that it allows you to scale without the complexity and administration overhead when it comes to connectivity.
+
+Today we will focus on how to deploy an AWS Multi-Account, Muli-VPC Transit Gateway Hub and Spoke Networking solution to integrate with your On-Premise network via a VPN connection.  Workload VPC's will be isolated from each other, meaning that each Workload VPC will only be allowed to communicate with the central "Networking" Account VPC and the On-Premise network. Workload VPC Subnets will all be private, with the Networking VPC hosting private and public subnets for Egress Internet traffic. 
+
+As a bonuse with Transit Gateway, we now get Centralized Monitoring with [Transit Gateway Network Manager](https://aws.amazon.com/transit-gateway/network-manager/) .  Providing centralized logs, Geo-location of your network connectivity  and realtime network topology maps showing connectivity status from VPN all the way down to the VPC level.  
+
+The Architecture we are going to deploy as code(Cloudformation) is illustrated below:
+
+![alt text](/img/TGW-Architecture.png)
+
+
+### Laying down the foundations needed to build the Networking Infrastructure
+The first step is to organize your AWS accounts with AWS Organizations, creating an Organization which will house the AWS accounts needed for your company to operate in the cloud. To configure this, login to your AWS Console Root/Master account and setup the account structure within an Organization.  As a minimum, you should have the following accounts members of the AWS Organization you create:
```

Comment:
You don't actually need an Org though, do you? I imagine most situations that are going to actually use this will - but if i just want to learn about transit gateway this is unnecessary, isn't it?